tests/integration/resources/import/pipeline_existing_jobs.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Can we store this file in `tests/integration/resources/import/pipeline/pipeline_existing_jobs.json` ?

Reply:
You're right, I will change that